fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/impl/content/registry/FuelItemRegistryImpl.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.content.registry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+
+import net.minecraft.block.entity.AbstractFurnaceBlockEntity;
+import net.minecraft.item.Item;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.fabric.api.content.registry.v1.util.ItemContentRegistry;
+
+public class FuelItemRegistryImpl extends ContentRegistryImpl<Item, Integer> implements ItemContentRegistry<Integer> {
+	public static final Map<Item, Integer> FUELS = new HashMap<>();
```

Comment:
private

Reply:
this should not be static when fuel item registry is already a singleton

Reply:
Make this an instance field. Otherwise there is no point for `INSTANCE` singleton below.